api/config/initializers/doorkeeper.rb

```diff
@@ -7,7 +7,7 @@
 
   resource_owner_from_credentials do
     user = Spree.user_class.find_for_database_authentication(email: params[:username])
-    user if user && user.valid_for_authentication? { user.valid_password?(params[:password]) }
+    user if user&.valid_for_authentication? { user.valid_password?(params[:password]) }
```

Comment:
I’m not sure  but did we decide to drop older ruby versions? If yes, never mind this comment :)

Reply:
@bbonislawski we've dropped 2.2 support some time ago ;)